.github/workflows/deploy-storefrontcloud.yml

```diff
@@ -49,16 +49,16 @@ jobs:
       - name: Deploy on vsf-next-demo.storefrontcloud.io
         if: github.ref == 'refs/heads/release/next'
         run: |
-          if curl -s -u ${{ secrets.CLOUD_USERNAME }}:${{ secrets.CLOUD_PASSWORD }} -H 'Content-Type: application/json' -X POST -d '{"code":"vsf-next-demo","frontContainerVersion":"${{ github.sha }}"}' https://farmer.storefrontcloud.io/instances | grep -q '{"code":200,"result":"Instance updated!"}'; then
+          if curl -s -H 'X-User-Id: ${{ secrets.DEMO_CLOUD_USERNAME }}' -H 'X-Api-Key: ${{ secrets.DEMO_CLOUD_PASSWORD }}' -H 'Content-Type: application/json' -X POST -d '{"code":"vsf-next-demo","frontContainerVersion":"${{ github.sha }}"}' https://farmer.storefrontcloud.io/instances | grep -q '{"code":200,"result":"Instance updated!"}'; then
             echo "Instance updated"
           else
             echo "Something went wrong during the update process..."
             exit 1
           fi
-      - name: Deploy on vsf-next-demo.europe-west1.gcp.storefrontcloud.io
-        if: github.ref == 'refs/heads/test/next'
+      - name: Deploy on demo-ct.europe-west1.gcp.storefrontcloud.io
```

Comment:
vsf-next-demo.europe-west1.gcp.storefrontcloud.io is used as a test env. Move it to additional step